lms/envs/devstack.py

```diff
@@ -261,6 +261,18 @@ def should_show_debug_toolbar(_):
     'JWT_AUDIENCE': 'lms-key',
 })
 
+# TODO: TNL-6546: Remove this waffle and flag code.
+from django.db.utils import ProgrammingError
+from waffle.models import Flag
```

Comment:
This breaks Docker devstack. I'm quite surprised it doesn't break Vagrant devstack. We should _never_ import models in settings since you have no clue when all settings have been loaded. In my case, the `SECRET_KEY` is not loaded at the time of this call. If data needs to be created, please do so via migration and/or provisioning.